packages/file-search/src/node/file-search-service-impl.spec.ts

```diff
@@ -195,4 +195,35 @@ describe('search-service', function (): void {
         });
     });
 
+    describe('search with whitespaces', () => {
+        it('should support file searches with whitespaces', async () => {
+            const rootUri = FileUri.create(path.resolve(__dirname, '../../test-resources')).toString();
+            const matches = await service.find('foo sub', { rootUris: [rootUri], fuzzyMatch: true, useGitIgnore: true, limit: 200 });
+
+            expect(matches).to.be.length(2);
+            expect(matches[0].endsWith('subdir1/sub-bar/foo.txt'));
+            expect(matches[1].endsWith('subdir1/sub2/foo.txt'));
+        });
+
+        it('should support non-fuzzy file searches with whitespaces', async () => {
+            const rootUri = FileUri.create(path.resolve(__dirname, '../../test-resources')).toString();
+            const matches = await service.find('foo sub', { rootUris: [rootUri], fuzzyMatch: false, useGitIgnore: true, limit: 200 });
+
+            expect(matches).to.be.length(2);
+            expect(matches[0].endsWith('subdir1/sub-bar/foo.txt'));
+            expect(matches[1].endsWith('subdir1/sub2/foo.txt'));
+        });
+
+        it('should support file searches with whitespaces regardless of order', async () => {
+            const rootUri = FileUri.create(path.resolve(__dirname, '../../test-resources')).toString();
+            const matchesA = await service.find('foo sub', { rootUris: [rootUri], fuzzyMatch: true, useGitIgnore: true, limit: 200 });
+            const matchesB = await service.find('sub foo', { rootUris: [rootUri], fuzzyMatch: true, useGitIgnore: true, limit: 200 });
+
+            expect(matchesA).to.be.length(2);
+            expect(matchesA.length).to.eq(matchesB.length);
+            expect(matchesA[0]).to.eq(matchesB[0]);
+            expect(matchesA[1]).to.eq(matchesB[1]);
```

Comment:
The following can be replaced with:

```suggestion
            expect(matchesA).to.deep.equal(matchesB);
```